python/pip_install/pip_hub_repository_requirements_bzlmod.bzl.tmpl

```diff
@@ -0,0 +1,32 @@
+"""Starlark representation of locked requirements.
+
+@generated by rules_python pip_parse repository rule
+from %%REQUIREMENTS_LOCK%%.
+
+This file is different from the other bzlmod template
+because we do not support entry_point yet.
+"""
+
+all_requirements = %%ALL_REQUIREMENTS%%
+
+all_whl_requirements = %%ALL_WHL_REQUIREMENTS%%
+
+def _clean_name(name):
+    return name.replace("-", "_").replace(".", "_").lower()
+
+def requirement(name):
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "pkg")
+
+def whl_requirement(name):
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "whl")
+
+def data_requirement(name):
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "data")
+
+def dist_info_requirement(name):
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "dist_info")
+
+def entry_point(pkg, script = None):
+    """entry_point returns the target of the canonical label of the package entrypoints.
+    """
+    print("not implemented")
```

Comment:
Nit, this could be in theory easily supported if we attempt to return a select statement pointing to the spoke repos. Probably out of scope of this PR.

Reply:
ref: https://github.com/bazelbuild/rules_python/issues/1262

Reply:
Yup, I found that after commenting here, LGTM.